internal/terraform/transform_targets.go

```diff
@@ -130,6 +138,41 @@ func (t *TargetsTransformer) selectTargetedNodes(g *Graph, addrs []addrs.Targeta
 	return targetedNodes
 }
 
+// addDependencies adds dependencies of the targeted vertex. This includes all ancestors in the
+// graph. It also includes all action trigger nodes in the graph. Actions are planned after the
+// triggering node has planned so that we can ensure the actions are only planned if the triggering
+// resource has an action (Create / Update) corresponding to one of the events in the action trigger
+// blocks event list.
+func (t *TargetsTransformer) addDependencies(g *Graph, v dag.Vertex, targetedNodes dag.Set, addrs []addrs.Targetable) {
```

Comment:
This seems odd to me - we already have dependency transformers on resources, so this method should only  add _action_ dependencies, correct?  If I'm reading it correctly, let's name this something clearer, like addTriggeredActionDependencies (addMaybeTriggeredActionDependencies?)

Reply:
Yeah at this point in the graph we have already added all dependencies and such. What this does is adding the dependencies we identify for the vertex (so the ancestors being what we added so far and the children that are action trigger related is what we are adding now) to the list of targeted nodes. I'll add a more descirptive name mentioning that the targeted nodes is being changed, I think that should help